src/libANGLE/Config.cpp

```diff
@@ -372,6 +372,17 @@ std::vector<const Config *> ConfigSet::filter(const AttributeMap &attributeMap)
                 case EGL_Y_INVERTED_NOK:
                     match = config.yInverted == static_cast<EGLBoolean>(attributeValue);
                     break;
+                case EGL_TEXTURE_SWIZZLING_TYPE:
```

Comment:
Can't we fit this request for channel order under `EGL_COLOR_BUFFER_TYPE` attribute? Using a new enum case, `EGL_BGR_BUFFER_ANGLE`. It seems like a more natural place for this, with fewer chance for conflicts with other attributes.